Document/0x04e-Testing-Authentication-and-Session-Management.md

```diff
@@ -568,9 +568,12 @@ For applications which require L2 protection, the MASVS states that: "The app in
 3. The application provides an overview of the last session after login at all times.
 4. The application has a self-service portal in which the user can see an audit-log and manage the different devices with which he can login.
 
-In all cases, you should verify whether different devices are detected correctly. Therefore, the binding of the application to the actual device should be tested. For instance: in iOS a developer can use `identifierForVendor` whereas in Android, the developer can use `Settings.Secure.ANDROID_ID` to identify an application instance.
+In all cases, you should verify whether different devices are detected correctly. Therefore, the binding of the application to the actual device should be tested. 
+In iOS, a developer can use `identifierForVendor`, which is related to the bundleID: the moment you change a bundle-ID, the method will return a different value. When the app is ran for the first time, make sure you store the String returned by `identifierForVendor` to the KeyChain, so that changes to it can be detected at an early stage.
```

Comment:
```suggestion
In iOS, a developer can use `identifierForVendor`, which is related to the bundle ID: the moment you change a bundle ID, the method will return a different value. When the app is ran for the first time, make sure you store the string returned by `identifierForVendor` to the KeyChain, so that changes to it can be detected at an early stage.
```

Reply:
Resolved a bit differently